nc models/error-handler.js

```diff
@@ -0,0 +1,15 @@
+exports.psqlErrorHandler = (err, request, response, next) => {
+    if (err.code === "22P02") {
```

Comment:
With TDD in mind, did you experience a 22P02 or write a test to check for a 400: Bad Request yet?